File: ss19.c
#include <stdio.h>
#include <stdlib.h>
int main ()
{
printf("Content-Type:text/html\n\n");
printf("<!DOCTYPE html>\n");
printf("<html><head>\n");
printf("<meta charset='UTF-8'><title>ss19.c Motta</title>\n");
printf("</head><body><a href='%s'>go back to %s </a>", getenv("HTTP_REFERER"), getenv("HTTP_REFERER"));


printf("</body></html>\n");
return 0;}

File: SS19.php
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title> ss19 php Motta </title>
</head>
<body>
	

<?php

$val1 = "click";
$val2 = "click me";

if (isset($_GET['text'])) {
	$val2 = $_GET['text'];
	$val2  .= "click";
	$val1 = $val2;
}
?> 

<a href="ss19.php?text=<?php echo $val1 ?>" name="mylink" ><?php echo $val2 ?></a>
    
   
</form>
</body>
</html>

